rust/operator-binary/src/main.rs

```diff
@@ -55,11 +73,11 @@ struct OpaRun {
 }
 
 #[tokio::main]
-async fn main() -> Result<(), error::Error> {
+async fn main() -> Result<(), Error> {
```

Comment:
We can go for anyhow here and remove the Error enum.
```suggestion
async fn main() -> anyhow::Result<()> {
```